src/gui/gui_land_combat.hpp

```diff
@@ -969,6 +969,59 @@ class counter_ico : public tinted_image_element_base {
 		if(visible)
 			tinted_image_element_base::render(state, x, y);
 	}
+
+	tooltip_behavior has_tooltip(sys::state& state) noexcept override {
+		return tooltip_behavior::variable_tooltip;
+	}
+	void update_tooltip(sys::state& state, int32_t x, int32_t y, text::columnar_layout& contents) noexcept override {
+		auto b = retrieve<dcon::land_battle_id>(state, parent);
+		dcon::regiment_id reg{};
+		switch(rank) {
+		case regiment_rank::attacker_front:
+			reg = state.world.land_battle_get_attacker_front_line(b).at(slot);
+			break;
+		case regiment_rank::attacker_back:
+			reg = state.world.land_battle_get_attacker_back_line(b).at(slot);
+			break;
+		case regiment_rank::defender_front:
+			reg = state.world.land_battle_get_defender_front_line(b).at(slot);
+			break;
+		case regiment_rank::defender_back:
+			reg = state.world.land_battle_get_defender_back_line(b).at(slot);
+			break;
+		}
+		if(reg) {
+			auto n = state.world.army_get_controller_from_army_control(state.world.regiment_get_army_from_army_membership(reg));
```

Comment:
do you want to handle the case where this could be a rebel regiment, even though they currently aren't in the game?